distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+
+* Before this update, the wrong port was exposed for `jaeger-production-query`, resulting in refused connection. With this update, the issue is fixed by exposing the Jaeger Query gRPC port (16685) on the Jaeger Query deployment. (link:https://issues.redhat.com/browse/TRACING-2968[TRACING-2968])
+
+* Before this update, when deploying Service Mesh on single-node OpenShift clusters in disconnected environments, the Jaeger Pod frequently went into the Pending state. With this update, the issue is fixed. (link:https://issues.redhat.com/browse/TRACING-3312[TRACING-3312])
+
+* Before this update, the Jaeger Operator Pod was restarting with the default memory value due to the *OOMKilled* error. With this update, this issue is fixed by removing the resource limits. (link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3173])
+
+=== Known issues
+
+* Apache Spark is not supported.
+ifndef::openshift-rosa[]
+
+* The streaming deployment via AMQ/Kafka is unsupported on IBM Z and IBM Power Systems.
+endif::openshift-rosa[]
+
+== {TempoName}
+
+:FeatureName: The {TempoName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+=== New features and enhancements
+
+This release introduces the following noteworthy enhancements for the {TempoShortName}:
+
+* Support the link:https://operatorframework.io/operator-capabilities/[operator maturity] Level IV, Deep Insights, which enables upgrading, monitoring, and alerting of `TempoStack` instances and the {TempoOperator}.
+
+* Add Ingress and OpenShift Route configuration for the Gateway.
+
+* Support managed and unmanaged states in the `TempoStack` custom resource.
+
+* Expose the following additional ingestion protocols in the Distributor service: Jaeger Thrift binary, Jaeger Thrift compact, Jaeger gRPC, and Zipkin. When the Gateway is enabled, only the OTLP gRPC is enabled.
+
+* Expose the Jaeger Query gRPC endpoint on the Query Frontend service.
+
+* Support multitenancy without Gateway authentication and authorization.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, the {TempoOperator} was not compatible with disconnected environments. With this update, the {TempoOperator} supports disconnected environments. (link:https://issues.redhat.com/browse/TRACING-3145[TRACING-3145])
+
+* Before this update, the {TempoOperator} with TLS failed to start on {product-title}. With this update, the mTLS communication is enabled between Tempo components, the Operand starts successfully, and the Jaeger UI is accessible. (link:https://issues.redhat.com/browse/TRACING-3091[TRACING-3091])
+
+* Before this update, the resource limits from the {TempoOperator} caused errors such as *OOMKilled*. With this update, the resource limits for the {TempoOperator} are removed to avoid such errors. (link:https://issues.redhat.com/browse/TRACING-3204[TRACING-3204])
```

Comment:
```suggestion
* Before this update, the resource limits from the {TempoOperator} caused error messages such as `reason: OOMKilled`. With this update, the resource limits for the {TempoOperator} are removed to avoid such errors. (link:https://issues.redhat.com/browse/TRACING-3204[TRACING-3204])
```

Reply:
Thank you. Applied manually. Memorizing error formatting.